doc/RELEASE-NOTES-0.2.0.md

```diff
@@ -0,0 +1,137 @@
+# RELEASE NOTES for "Good Peers" sprint
+
+The 3rd sprint of our development focused on the following issues:
+* ready all objects in the system for network exchange: define canonical form and serialization
+* build sync infrastructure
+* further work on internal state structure: Authenticated Dictionaries
+* further research on consensus architecture
+
+The packages for the this release are available at the following location:
+https://github.com/aeternity/epoch/releases/tag/v0.2.0-good-peers
+
+To fully appreciate the newest developments you need to experience syncing nodes and sharing blocks.
+
+The easiest way to achieve it, is to deploy 3 local nodes and see how they communicate.
+Please find detailed information about custom configuration here:
+https://github.com/aeternity/epoch/blob/4d1baf36207dfeff2d2f1a5256d37285574b7457/doc/CUSTOMIZATION.md
+
+1. Download packages for Good Peers release, unpack and enter the directory.
+
+2. Make total 3 copies and adjust VM config
+
+Please use following sed command - tested on OSX and on Ubuntu (as per Vagrant box "ubuntu/trusty64"):
+
+* For node 1 (when in the top release dir):
+  ```
+  sed -ibkp 's/-sname epoch/-sname epoch1/g' ./releases/0.2.0/vm.args
+  ```
+
+* For node 2 (when in the top release dir):
+  ```
+  sed -ibkp 's/-sname epoch/-sname epoch2/g' ./releases/0.2.0/vm.args
+  ```
+
+* For node 3 (when in the top release dir):
+  ```
+  sed -ibkp 's/-sname epoch/-sname epoch3/g' ./releases/0.2.0/vm.args
+  ```
+
+3. Create following files in the top release dir:
+
+In node 1, file:
+
+p1.config:
+```
+[
+{aecore,
+ [
+  {peers, ["http://localhost:9902/",
+           "http://localhost:9903/"]}
+ ]},
+{aehttp,
+ [
+  {swagger_port_external, 9900}
+ ]}
+].
+```
+
+In node 2, file:
+
+p2.config
+```
+[
+{aecore,
+ [
+  {peers, []}
+ ]},
+{aehttp,
+ [
+  {swagger_port_external, 9902}
+ ]}
+].
+```
+
+In node 3, file:
+
+p3.config
+```
+[
+{aecore,
+ [
+  {peers, []}
+ ]},
+{aehttp,
+ [
+  {swagger_port_external, 9903}
+ ]}
+].
+```
+
+4. Use configs supplied below to start nodes. Start nodes (in order 3. 2. 1.)
```

Comment:
configs now are above I think. Or by configs do we mean the command lines?

Reply:
well spotted

Reply:
I am rephrasing

Reply:
Rephrased and force-pushed.